Content.Shared/Nutrition/Components/HungerComponent.cs

```diff
@@ -41,6 +42,7 @@ public sealed partial class HungerComponent : Component
     /// The current hunger threshold the entity is at
     /// </summary>
     [DataField("currentThreshold"), ViewVariables(VVAccess.ReadWrite)]
+    [AutoNetworkedField]
```

Comment:
Why did it become the only networked field for hunger?

Reply:
Because it's the only field I'm reading.

Reply:
because system is in shared and is run on both client and server, all fields are needed to be auto networked so we don't have prediction issues on client. Just put [AutoNetworkedField] on all fields from states you removed.